### PR TITLE
test: DAH-2974 fix test so that it uses a fcfs rental listing

### PR DESCRIPTION
**Pull request by tallulahkay**
DAH-2974

### CODE REVIEW

File: cypress/e2e/ApplicationNewPage.e2e.js [hunk 1]
```diff
@@ -35,303 +35,303 @@ describe('ApplicationNewPage', () => {
...
-    // Select live vs work
-    cy.get('#form-preferences\\.0\\.individual_preference').select('Live in SF').blur()
-
-    // Verify that preference proof has been unset and is validated.
-    cy.get('.save-btn').click()
-    cy.contains('Type of Proof is required').should('exist')
-
-    // Select type of proof
-    cy.get('#form-preferences\\.0\\.type_of_proof').select('Telephone bill').blur()
+  // it('should fail if required fields are missing, and create application if not', () => {
```
Comment (alulabeshue-sfgov) [inline cypress/e2e/ApplicationNewPage.e2e.js]: suggestion: can you add to the ticket or this PR why we are commenting out tests?

File: cypress.config.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ module.exports = defineConfig({
       // Default to sample flagged record set on full
       FLAGGED_RECORD_SET_ID: process.env.E2E_FLAGGED_RECORD_SET_ID || 'a0r0P00002WqGZ6QAN',
       // Default to 400 China Basin in full
-      FCFS_LISTING_ID: process.env.FCFS_LISTING_ID || 'a0W4U00000Nlb5lUAB',
+      FCFS_LISTING_ID: process.env.FCFS_LISTING_ID || 'a0W6s000008CO5yEAG',
```
Comment (alulabeshue-sfgov) [inline cypress.config.js]: suggestion: rename this constant to include rental in the name so it is clear the kind of fcfs listing

**Review by alulabeshue-sfgov — COMMENTED** [review]
Failing test now passes after the change to a rental fcfs listing. Didnt understand why we are commenting out tests that were passing.

File: cypress/e2e/ApplicationNewPage.e2e.js (hunk 1 above)
Comment (tallulahkay) [reply cypress/e2e/ApplicationNewPage.e2e.js]: I was just doing that to make it faster to run locally! sorry I'll uncomment these

File: cypress.config.js (hunk 2 above)
Comment (tallulahkay) [reply cypress.config.js]: love it!

**Review by alulabeshue-sfgov — APPROVED** [review]
LG, thanks!

**Pull request merged**